### ISSUE TITLE
Muilti-label-seg_bug+report

### ISSUE THREAD

**Issue by jinxsfe** [HenriquesLab/ZeroCostDL4Mic#331]
![image](https://github.com/user-attachments/assets/60246e0e-0352-4ead-b33b-bde3a5c4d9de)
I tried to install dependency for multilabel UNET2D but the error show there

**Comment by IvanHCenalmor** [comment]
Hi @jinxsfe,

The U-Net 2D multilable notebook has been recently updated to load and export models for the [BioImage Model Zoo](https://bioimage.io/#/). In order to be able to export the models, we have change the TensorFlow and Keras dependencies. This has created the dependency issue that you are getting. 

Still, this is not a real problem for the notebook. We have tested it and besides that dependency issue it runs without any problem.

Please try to ignore that message and continue running the notebook as normal. If you find another issue or bug while running that notebook, please let us know and we will try to fix it as fast as possible.

Thank you very much for your contribution! ❤️ 

Best,
Iván
